modmail/utils/pagination.py

```diff
@@ -251,9 +239,18 @@ def update_states(self) -> None:
         if the paginator is on the last page, the jump last/move forward buttons will be disabled.
         """
         # update the footer
-        text = self.get_footer()
-        if self.embed:
-            self.embed.set_footer(text=text)
+        if self.embed is None:
+            self.content = self._pages[self.index]
+        else:
+            self.embed.description = self._pages[self.index]
+            page_indicator = f"Page {self.index+1}/{len(self._pages)}"
+            self.embed.set_footer(
+                text=(
+                    f"{self.footer_text} ({page_indicator})"
+                    if self.footer_text is not None
+                    else page_indicator
+                )
+            )
```

Comment:
This is awkward, can we un-one-line this? We can also not compare with None, as setting footer text to "" will result in not great output anyway, so we just check if it's present.
```py
if self.footer_text:
    footer_text = f"{self.footer_text} ({page_indicator})"
else:
    footer_text = page_indicator
```

Reply:
How is this already outdated when no commits have been made? 🤔

Reply:
implemented in c9b33a3